FRPs/proposal-2-liquidations.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 2
+title: MEV and Liquidations
+team: Kaihua Qin, Liyi Zhou, Arthur Gervais
+created: 2020-12-10
+paper: 1
+---
+
+# Summary
+With over 6B USD total value locked for the top 3 lending protocols alone, this use case has grown into a substantial DeFi market. Lending entails the risks of a diminishing collateral value, which may render debt unhealthy. Unhealthy debt is typically resolved through liquidations, a process of selling collateral at a discount to liquidators.
+
+One dominant liquidation design is the fixed spread liquidation mechanism, used by Aave, Compound and dYdX. Interestingly, the fraction of the collateral that a liquidator can purchase at a discount amounts to 50% for Aave and Compound, and even 100% for dYdX (commonly referred to as close factor). It therefore appears that the fixed spread liquidation mechanisms are benefitting the liquidators more than the borrowers.
+
+Drawing the link to MEV, each liquidation event may qualify as Miner Extractable Value (MEV). For example, in a recent price oracle manipulation attack, a liquidator performed an oracle price post and liquidated millions of USD within one transaction, yielding a profit of nearly 4M USD. 
+
+The goal of this FRP is to offer a systematization of existing liquidation mechanisms and consistent measurements of the past and current DeFi liquidation state. We aim to investigate alternative liquidation designs in an effort to mitigate MEV from liquidation events.
+
+## Background and Problem Statement
+
+The goal of this FRP is to study the past and current status of blockchain liquidation. Specifically, we would like to answer the following questions:
+- What is the monthly accumulated liquidation profit for Aave, Compound, dYdX, MakerDAO?
+- What gas price distributions are being paid for the liquidation events?
```

Comment:
MEV Inspect surfaces that info for Aave and Compound, you may want to add inspectors for dydx and MakerDAO. I sense that uncovering the nuanced details of MakerDAO liquidations might be harder, due to them happening on a multi transaction basis.

Reply:
Thank you for the pointer.

Reply:
I would also like more info on how this could go beyond MEV inspect to perhaps study complex bot behaviors beyond simple liquidations (eg through custom business logic in proxy contracts)?  My primary concern is how far beyond MEV inspect we can go and what the limitations are of studying limitations without quantifying such business logic.

Reply:
Question: @gakonst how does MEV Inspect handle metatransactions? Do you introspect into flash loans?